Example/Source/Examples/Layout/OffsetBrickViewController.swift

```diff
@@ -24,15 +24,15 @@ class OffsetBrickViewController: BaseRepeatBrickViewController {
 }
 
 extension OffsetBrickViewController: OffsetLayoutBehaviorDataSource {
-    func offsetLayoutBehavior(behavior: OffsetLayoutBehavior, originOffsetForItemAtIndexPath indexPath: NSIndexPath, withIdentifier identifier: String, inCollectionViewLayout collectionViewLayout: UICollectionViewLayout) -> CGSize? {
+    func offsetLayoutBehaviorWithOrigin(_ behavior: OffsetLayoutBehavior, originOffsetForItemAtIndexPath indexPath: IndexPath, withIdentifier identifier: String, inCollectionViewLayout collectionViewLayout: UICollectionViewLayout) -> CGSize? {
```

Comment:
if we already have `withOrigin` in the function name, do we still need it in the variable name?
Maybe follow the standard what UITableView is doing
`func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {`
so that would be 
`func offsetLayoutBehavior(_ behavior: OffsetLayoutBehavior, originOffsetForItemAt indexPath: IndexPath, with identifier: String, in collectionViewLayout: UICollectionViewLayout) -> CGSize? {`